scripts/release.py

```diff
@@ -261,12 +261,23 @@ def add_release_to_changelog(name: str, release_notes: str):
 
 def commit_and_push(dd_repo, branch_name: str, release_name: str):
     subprocess.check_output(f"git checkout -b {branch_name}", shell=True, cwd=os.pardir)
-    subprocess.check_output("git add -A", shell=True, cwd=os.pardir)
+    try:
+        subprocess.check_output("git add CHANGELOG.md", shell=True, cwd=os.pardir)
```

Comment:
`git add $(git rev-parse --show-toplevel)/CHANGELOG.MD)`

I was just curious if you could get the root directory of the repo regardless of which sub-directory you might be in.

Reply:
I am not sure `$(command)` works in Python subprocess though....

Reply:
Yeah I think for this to work you'd need to create multiple subprocesses and pipe them together. I like the flexibility of that approach, though. Possible future improvement.

Reply:
@ZStriker19 maybe line 36 in the docstring of this file should change if it can be run from cwds other than `scripts`.